.github/workflows/maven.yml

```diff
@@ -0,0 +1,68 @@
+name: Arquillian-Cube Maven Build
+
+on:
+  push:
+    branches:
+      - '**'
+  pull_request:
+    branches:
+      - '**'
+env:
+  # failsafe.groups configuration depends on the env setup.
+  FAILSAFE_GROUPS: ''
+
+jobs:
+  build:
+    runs-on: ${{ matrix.os }}
+    timeout-minutes: 90
+    strategy:
+      fail-fast: false
+      matrix:
+        os: [ubuntu-latest]
+        java: ['8']
+
+    steps:
+      - name: Setup Minikube
+        uses: manusa/actions-setup-minikube@v2.9.0
+        with:
+          driver: docker
+          container runtime: containerd
+          minikube version: 'v1.30.1'
+          kubernetes version: 'v1.27.1'
+          github token: ${{ secrets.GITHUB_TOKEN }}
+          start args: "--memory='4gb' --cpus='2'"
```

Comment:
Any specific requirement that we might want to document with a comment here?

Reply:
Do you mean the args to start the minikube ?   By default minikube is started with `2 cpus` and `2gb memory` and it's not enough for some heavy project, so I explicitly added these arguments to minikube.

Reply:
Perfect, thanks.